src/masoniteorm/schema/Blueprint.py

```diff
@@ -709,6 +709,21 @@ def primary(self, column=None):
 
         return self
 
+    def add_foreign(self, columns: string):
+        """Creates the foreign spliting the foreign name, reference column, and
+        reference table.
+
+        Arguments:
+            columns {string} -- The name of the from_column . to_column . table
+        """
+        columns_array: list = columns.split(".")
```

Comment:
I found using unpacking is more clear.

Changing this line to 
```python
        from_column, to_column, table = columns.split(".")
```

makes things a little more clear on what is going on

Reply:
And remove the validation down on leve it ?